<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QWidget" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>335</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>335</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BLauncher v2.01  - Minecraft Windows 10 Edition</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/logo.ico</normaloff>assets/logo.ico</iconset>
  </property>
  <widget class="QPushButton" name="startb">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Activate Exploit and Launch Game.</string>
   </property>
   <property name="text">
    <string>Start Playing</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>421</width>
     <height>68</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Keep adjusting this slider till you get desired results.</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSlider" name="slider1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>340</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="sliderPosition">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>37</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>7</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Set Initial Kill Timer :-                  (seconds)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>421</width>
     <height>68</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Decrease its value if your game becomes Trail while playing.</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSlider" name="slider2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>340</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
    <property name="sliderPosition">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>37</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>30.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>7</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Set Periodic Kill Timer :-                (minutes)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>17</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="BL_LOGO">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>8</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap>assets/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="fix2b">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>125</width>
     <height>35</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Use this button if you are unable to Install any apps from Microsoft Store.</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Store Fix</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>assets/configure.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fix1b">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>120</y>
     <width>125</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use this button if your game crashes while playing.</string>
   </property>
   <property name="text">
    <string>Crash Fix</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>assets/configure.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QSpinBox" name="notif_duration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>676</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="DISCORD">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>290</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>assets/discord_join.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>true</strikeout>
    </font>
   </property>
   <property name="text">
    <string>• Notification Duration  (seconds) :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YOUTUBE">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>290</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>assets/yt_sub.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>NOTE : You need to Turn off &quot;Focus Assist&quot; in windows settings to recieve in-game notifications.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="notif_agree">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>You will recieve a Notification 30 seconds before any potential Crash may Occur.
Focus Assist may mute the notifications so make sure to turn off Focus Assist in Windows Settings :)</string>
   </property>
   <property name="text">
    <string>Get Potential Crash Alert (via Notification)</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="dev_info">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>10</y>
     <width>125</width>
     <height>101</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>6</y>
      <width>125</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string extracomment="Visit BLauncher Website"/>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://techarchives.rf.gd&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#ffff00;&quot;&gt;BLauncher&lt;br/&gt;v2.0&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>• Theme :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>51</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>75</y>
     <width>56</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>• Author :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="auth_name">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>75</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="optionb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>131</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>More Options !</string>
   </property>
   <property name="text">
    <string>Show Options</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../../../../.designer/backup/assets/configure.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="consoleb">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>131</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>More Options !</string>
   </property>
   <property name="text">
    <string>Show Console</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../../../../.designer/backup/assets/configure.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="console">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>342</y>
     <width>420</width>
     <height>90</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>710</width>
     <height>90</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>startb</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>fix2b</zorder>
  <zorder>fix1b</zorder>
  <zorder>notif_duration</zorder>
  <zorder>DISCORD</zorder>
  <zorder>label_2</zorder>
  <zorder>YOUTUBE</zorder>
  <zorder>label_6</zorder>
  <zorder>notif_agree</zorder>
  <zorder>dev_info</zorder>
  <zorder>label_8</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_9</zorder>
  <zorder>auth_name</zorder>
  <zorder>optionb</zorder>
  <zorder>consoleb</zorder>
  <zorder>console</zorder>
 </widget>
 <tabstops>
  <tabstop>startb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
